vega_sim/scenario/configurable_market/scenario.py

```diff
@@ -152,6 +153,20 @@ def configure_agents(
             state_update_freq=10,
         )
 
+        at_the_touch_mm = AtTheTouchMarketMaker(
+            key_name="AT_THE_TOUCH_MM",
+            wallet_name="MM",
+            initial_asset_mint=1e9,
+            market_name=self.market_name,
+            asset_name=self.asset_name,
+            market_decimal_places=market_config.decimal_places,
+            position_decimal_places=market_config.position_decimal_places,
+            asset_decimal_places=self.asset_decimal,
+            peg_offset=30 * 10 ** (-market_config.decimal_places),
```

Comment:
If we can set `peg_offset` in both this and below to `0` I think we're good to go